source/change-streams/tests/unified/change-streams.yml

```diff
@@ -0,0 +1,74 @@
+description: "change-streams"
+schemaVersion: "1.0"
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: []
+      ignoreCommandMonitoringEvents: []
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: *database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: *collection0
+initialData:
+  - collectionName: *collection0
+    databaseName: *database0
+    documents: []
+tests:
+  - description: "Test array truncation"
+    runOnRequirements:
+      - minServerVersion: 4.7
+        topologies: [replicaset]
+    operations:
+      - name: insertOne
+        object: *collection0
+        arguments:
+          document: {
+            "_id": 1,
+            "a": 1,
+            "array": ["foo", {"a": "bar"}, 1, 2, 3]
+          }
+      - name: createChangeStream
+        object: *collection0
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: updateOne
```

Comment:
This `updateOne` operation is not supported by the unified test runner in the Java driver. @divjotarora Helped me with this operation and confirmed it is supported by the Go driver.

The reason this can't be a simple `findOneAndUpdate` operation is that the MongoDB server currently may consider using `truncatedArrays` only if the update is done via an aggregation pipeline.

@divjotarora Could you please try the committed version of the test with the Go driver to make sure it still works? (I cannot currently run the test with the Java driver).

Reply:
You should add support for updateOne in the Java driver's unified test runner and confirm that the test passes.

Reply:
Yes, done in https://github.com/mongodb/mongo-java-driver/pull/667.

Reply:
Ack.  Thanks!

Reply:
The test currently fails in Go (see my comment on the `createChangeStream` operation above). If I fix that issue locally, though, the test passes.

Reply:
Done. I also updated the unified runner in the Java driver.